package.json

```diff
@@ -111,9 +111,13 @@
     "yargs": "6.6.0"
   },
   "dependencies": {
+    "angular-bootstrap-switch": "^0.5.1",
+    "angular-dragdrop": "^1.0.13",
+    "angular-ui-sortable": "^0.16.1",
     "express": "4.14.0",
     "http-proxy": "1.16.2",
-    "manageiq-ui-components": "0.0.11",
+    "jquery-ui-bundle": "^1.11.4",
+    "manageiq-ui-components": "romanblanco/ui-components#dialog_editor",
```

Comment:
The idea behind `ui-components` is that any MiQ application can use some kind of this library so some shared component (dialog editor, table, tile...) will behave same across all applications. However you don't need to use every component which is in `ui-components`, you might use 5 in SUI 7 in `ui-classic` etc. For this purpose we did not added any 3rd party library to `ui-components`. So if you want to use dialog editor from `ui-components` you have to install 3rd party libraries which are required by components corresponded for dialog editor (`jquery-ui`, `bootstrap-switch`, `angular-bootstrap-switch`, `angular-ui-sortable`, `angular-dragdrop`), but if you are not using these components you don't have to install these libraries.